core/src/main/java/com/sequenceiq/cloudbreak/reactor/api/event/stack/loadbalancer/handler/CreateLoadBalancerEntityHandler.java

```diff
@@ -114,10 +118,13 @@ protected Selectable doAccept(HandlerEvent<CreateLoadBalancerEntityRequest> even
                 measure(() -> instanceGroupService.saveAll(newLoadBalancers.stream()
                         .flatMap(lb -> lb.getAllInstanceGroups().stream()).collect(Collectors.toSet())),
                     LOGGER, "Instance groups saved in {} ms for stack {}", stackName);
+                measure(() -> networkService.pureSave(stack.getNetwork()), LOGGER,
```

Comment:
This won't be affected by the network being null right? I assume not but just making sure since Yarn has a null network field for its stack objects.

Reply:
Good call. I'll put a null check around it.